components/linkButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {Linking, StyleSheet, Text, View} from 'react-native';
+import {Colors, GlobalStyles} from '../styles';
+
+export default function LinkButton({title, url}) {
+  return (
+    <View style={styles.content}>
+      <Text style={styles.text} onPress={() => {Linking.openURL(url)} }>{title}</Text>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  content: {
+    ...GlobalStyles.content,
```

Comment:
I wouldn't include the higher-level content-level styling in the component itself. Let the parent of the component (i.e. the screen) handle what the sizing/spacing/margins should be for and around the button. It makes it more re-usable in other contexts.